assets/pages/paper-subscription-landing/components/content.jsx

```diff
@@ -0,0 +1,146 @@
+                'Your newsagent won’t lose out; we’ll pay them the same amount that they’d receive if you paid cash for your paper',
+                'You can pause your subscription for up to four weeks a year. So if you’re going away, you won’t have to pay for the papers that you miss.',
+              ]}
+            />
+          </ProductPageTextBlock>
+        </ProductPageContentBlock>
+        <ProductPageContentBlock type="feature">
+          <ProductPageTextBlock title="Now pick your perfect voucher subscription package" />
+          <ProductPageContentBlockDivider />
+          <ProductPageTextBlock>
+            <p>When you take out a voucher subscription, we’ll send you a book of vouchers.
+              There’s one for each newspaper in the package you choose. So if you choose a
+              Sixday package, for example, you’ll receive six vouchers for each week,
+              delivered every quarter
+            </p>
+          </ProductPageTextBlock>
+          <Form />
+        </ProductPageContentBlock>
+        {faqs}
+      </div>
+    );
+
+    const linkToVouchers = (
+      <a
+        href={paperSubsUrl('collection')}
+        onClick={(ev) => {
+        ev.preventDefault();
+        setTabAction('collection');
+      }}
+      >
+      subscribe using our voucher scheme
+      </a>);
+
+    const deliveryPage = (
+      <div className="paper-subscription-landing-content__focusable" tabIndex={-1} ref={(d) => { this.tabRef = d; }}>
+        <ProductPageContentBlock>
+          <ProductPageTextBlock title="How does delivery work?">
+            <ProductPageTextBlockList items={[
+                  (
+                    <span>
+                    If you live in Greater London (within the M25),
+                    you can use The Guardian’s home delivery service. Don’t
+                    worry if you live outside this area - you can still {linkToVouchers}.
+                    </span>
+                  ),
+                  'Your paper will arrive before 7am from Monday to Saturday and before 8.30am on Sunday',
+                  'Your newspaper deliveries will begin as quickly as five days from you subscribing',
+                  'Delivery to individual flats, or apartments within blocks, is unavailable as we require access to your post box to deliver your paper',
+                  'You can pause your subscription for up to 36 days a year. So if you’re going away anywhere, you won’t have to pay for the papers that you miss',
+                ]}
+            />
+          </ProductPageTextBlock>
+        </ProductPageContentBlock>
+        <ProductPageContentBlock type="feature">
+          <ProductPageTextBlock title="Now pick your perfect delivery subscription package" />
+          <ProductPageContentBlockDivider />
+          <Form />
+        </ProductPageContentBlock>
+        {faqs}
+      </div>
+    );
```

Comment:
Can these be broken out into functional components using the "Auxiliary Components" pattern seen elsewhere in the codebase?